src/test/java/edu/cornell/kfs/pdp/batch/fixture/StandardEntryClassConversionFixture.java

```diff
@@ -0,0 +1,62 @@
+package edu.cornell.kfs.pdp.batch.fixture;
+
+import org.kuali.kfs.pdp.businessobject.options.StandardEntryClassValuesFinder;
+import edu.cornell.kfs.coa.businessobject.options.CuCheckingSavingsValuesFinder;
+
+public class StandardEntryClassConversionFixture {
+    
+    public enum ExpectFailureData {
+        NULL_TEST("NULL_TEST", null),
+        BLANK_TEST("BLANK_TEST", ""),
+        WHITESPACE_TEST("WHITESPACE_TEST", "        "),
+        SINGLE_CHAR_TEST("SINGLE_CHAR_TEST", "P"),
+        TWO_CHAR_TEST("TWO_CHAR_TEST", "PD"),
+        INVALID_THREE_CHAR_TEST("INVALID_THREE_CHAR_TEST", "ABC"),
+        INVALID_FOUR_CHAR_TEST("INVALID_FOUR_CHAR_TEST", "PPD2");
+    
+        public final String typeOfTest;
+        public final String achBankAccountTypeBeingTested;
+        
+        private ExpectFailureData(String typeOfTest, String achBankAccountTypeBeingTested) { 
+            this.typeOfTest = typeOfTest;
+            this.achBankAccountTypeBeingTested = achBankAccountTypeBeingTested;
+        }
+    }
+    
+    public enum ExpectPDPSuccessData {
+        WORKDAY_PERSONAL_CHECKING_TEST("WORKDAY_PERSONAL_CHECKING_TEST", "22PPD", StandardEntryClassValuesFinder.StandardEntryClass.PPD.toString()),
+        WORKDAY_PERSONAL_SAVINGS_TEST("WORKDAY_PERSONAL_SAVINGS_TEST", "32PPD", StandardEntryClassValuesFinder.StandardEntryClass.PPD.toString());
```

Comment:
If you want to avoid having to call `toString()` on each applicable parameter, you could defer the `toString()` call to the constructor.